src/eei.h

```diff
@@ -79,6 +79,109 @@ struct EthereumInterface : ShellExternalInterface {
   void memoryCopy(std::vector<char> const& src, uint32_t srcoffset, uint32_t dstoffset, uint32_t length);
   void copyAddressToMemory(struct evm_address const& address, uint32_t dstoffset);
 
+  /*
+   * Memory Operations
+   */
+  
+  void loadMemory(uint32_t srcOffset, uint8_t *dst, size_t length)
+  {
+      assert(dst != nullptr);
+      assert(length);
+      
+      for (uint32_t i = 0; i < length; ++i) {
+          *(dst + i) = memory.get<uint8_t>(srcOffset + i);
+      }
+  }
+
+  void storeMemory(uint32_t dstOffset, const uint8_t *src, size_t length)
+  {
+      assert(src != nullptr);
+      assert(length);
+
+      for (uint32_t i = 0; i < length; ++i) {
+          memory.set<uint8_t>(dstOffset + i, *(src + i));
+      }
+  }
+
+  /*
+   * Memory Op Wrapper Functions
+   */
+
+  void loadUint256(uint32_t srcOffset, struct evm_uint256be *dst)
```

Comment:
Just return `evm_uint256be`. Similarly in other helpers.

Reply:
done on the helper functions branch now.